jax/_src/scipy/stats/_core.py

```diff
@@ -207,9 +207,9 @@ def sem(a: ArrayLike, axis: int | None = 0, ddof: int = 1, nan_policy: str = "pr
     b = b.ravel()
     axis = 0
   if nan_policy == "propagate":
-    return b.std(axis, ddof=ddof) / jnp.sqrt(b.shape[axis]).astype(b.dtype)
+    return b.std(axis, ddof=ddof, keepdims=keepdims) / jnp.sqrt(b.shape[axis]).astype(b.dtype)
```

Comment:
When `axis=None`, `b.shape[axis]` will have to be `b.size` instead.

Reply:
Thanks! Modified accordingly.